config/v1/types_infrastructure.go

```diff
@@ -182,6 +182,25 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// provisioningInterface is the name of the network interface on a Baremetal server connected
+	// to the provisioning network.
+	ProvisioningInterface string `json:"provisioningInterface"`
+
+	// provisioningIp is the VIP to be used by the provisioning service. This must be on the provisioning
+	// subnet, and outside of the DHCP range.
+	ProvisioningIp string `json:"provisioningIp"`
+
+	// dhcpRange is the IP address range on the provisioning subnet from which Baremetal hosts
+	// can be assigned an IP address.
+	DhcpRange string `json:"dhcpRange"`
```

Comment:
if this is a CIDR, we use CIDR in the network API to describe this.

Reply:
DHCP

Reply:
@deads2k  Current;y, it is comma separated pair of IP addresses where the 1st IP is the start of the range and the 2nd IP represents the end of the range. I can update the description to make this clear.